exotic/exotic.py

```diff
@@ -2686,7 +2688,7 @@ def main():
                         arrayAirmass = airMassList[~filtered_data]
 
                         # remove nans
-                        nanmask = np.isnan(arrayFinalFlux) | np.isnan(arrayNormUnc) | np.isnan(arrayTimes) | np.isnan(arrayAirmass) | arrayFinalFlux > 0 | arrayNormUnc > 0
+                        nanmask = np.isnan(arrayFinalFlux) | np.isnan(arrayNormUnc) | np.isnan(arrayTimes) | np.isnan(arrayAirmass) | np.greater(arrayFinalFlux, 0) | np.greater(arrayNormUnc, 0)
```

Comment:
the two `np.greater` should actually be:

`(arrayFinalFlux <= 0) | (arrayNormUnc <= 0)`

as it flags values that are less than 0

Reply:
That would make `nanmask` all `False` if the values are above`0` unless I'm mistaken. Aren't we filtering for values less than or equal to `0`, making those ones `False`?

Reply:
the nanmask is True for values that need to be filtered out, so you need to ID which values are <= 0

np.isnan returns True if the value is a nan, and therefore needs to be filtered out

Reply:
Oh gotcha, makes sense. Thought `False` would be the ones filtered out since I've done that before. Makes it much more clear when you mention `np.isnan` into this. Thanks!

Reply:
if you look elsewhere in the code, @pearsonkyle then filters out these bad values via:

`filtered_array = array[~nanmask]`

for the record, I also do it the other way where True == value is kept, but @pearsonkyle likes to dance to the beat of his own bongo :P